.github/workflows/ci_linux.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         #
         #         apt-get install python3-venv
         #
-        sudo apt -y install libsdl2-dev python3-venv valgrind spirv-tools glslang-dev libvulkan-dev
+        sudo apt -y install libsdl2-dev python3-venv valgrind spirv-tools glslang-dev libvulkan-dev mesa-vulkan-drivers
```

Comment:
unrelated but since you're at it: maybe add `spirv-tools glslang-dev libvulkan-dev mesa-vulkan-drivers` to the coverage CI as well? It's disabled though currently…

Also: you may want to add `vk` to `--debug-opts`

Reply:
Done.